site/docs/samples/chaining-sample.md

```diff
@@ -13,7 +13,7 @@ To recreate this scenario, you will need a copy of the Fybrik repository (`git c
     cd $FYBRIK_DIR
     go run main.go taxonomy compile --out custom-taxonomy.json --base charts/fybrik/files/taxonomy/taxonomy.json $AIRBYTE_MODULE_DIR/fybrik/fybrik-taxonomy-customize.yaml
     helm install fybrik-crd charts/fybrik-crd -n fybrik-system --wait
-    helm install fybrik charts/fybrik --set global.tag=master --set global.imagePullPolicy=Always -n fybrik-system --wait --set-file taxonomyOverride=custom-taxonomy.json
+    helm install fybrik charts/fybrik --set coordinator.catalog=openmetadata --set openmetadataConnector.openmetadata_endpoint=http://openmetadata.open-metadata:8585/api -n fybrik-system --wait --set-file taxonomyOverride=custom-taxonomy.json
```

Comment:
all commands assume that we work with master. 
I'd suggest adding:
git checkout {{ currentRelease|default('master') }}

Reply:
`openmetadataConnector.openmetadata_endpoint=http://openmetadata.open-metadata:8585/api` is the default value so I think it can be omitted.

Reply:
True. However, I suggest that we keep it, to help users who want to use a different OpenMetadata instance (e.g., outside kubernetes).